app/views/patron/_patron.html.erb

```diff
@@ -10,6 +10,10 @@
     <dt class="col-2 col-sm-1 font-weight-normal">Borrower limit:</dt>
     <dd class="col-10 col-sm-11 patron-status"><%= pluralize(patron.borrow_limit, 'checkout') %></dd>
   <% end %>
+  <% if current_user.library_id_user? %>
```

Comment:
Per @saseestone: we should tie this to `patron.fee_borrower?`, not how they logged in.

Reply:
That simplifies things :) ! 

I've pushed up the change and removed the extra logic I added.